Notes

```diff
@@ -1,9 +1,9 @@
 # To build:
-autoreconf -i
-CC="gcc-4.8.3" CFLAGS="-fms-extensions -std=c11" ./configure && make
-# You shouldn't need that exact version of gcc, it just has to be recent and
-# you shouldn't really need the -fms-extensions either if yours is up to date
-# enough.
+mkdir build
```

Comment:
You could just whap a `.gitignore` in a pre-made directory to stop everyone having to do this. See [this popular StackOverflow answer](https://stackoverflow.com/questions/115983/how-can-i-add-an-empty-directory-to-a-git-repository)

Reply:
That's what I did for rage, and it was sort of good, and sort of really annoying.
When you want to clear your build directory you can't just delete it with impunity due to the gitignore.